### ISSUE TITLE
Idea: Automate client script

### ISSUE THREAD

**Issue by kane-mason** [thomas4019/expressa#90]
While i don't think it will necessarilly be quick, i do think it would be relatively straightforward to generate a javascript client script containing all the default calls to all the collections. 

Something similar to what is exposed by expressa in node ie: `api.db.users.get` but for the endpoints specified in the the readme. The idea being the client script does the more laborious url construction and the consumer on the client just does something like `api.users.get( ... )` or `await api['email_signups].update( ... )` etc. 

Exposing all the collections is probably not a great idea, so a whitelisting approach could be used by setting a property on the collection schema 'for client'. Client script would then be generated when the database is created and when collections are added/removed.

Just an idea!

**Comment by thomas4019** [comment]
Yeah, this is something which makes frontend dev a little simpler and safer. Another developer made a solution that worked with an older version of expressa, https://github.com/coderofsalvation/expressa-client. Eventually I want to rework it a bit and bring it into this repo.

**Comment by kane-mason** [comment]
Thanks!
